index.test.js

```diff
@@ -1,22 +1,22 @@
 "use strict";
 
+const { ESLint } = require("eslint");
+
 const config = require("./");
 
-it("test basic properties of config", function() {
+it("test basic properties of config", function () {
```

Comment:
is that a jest 'thing' to have a whitespace in between `function` and `(` ?

Reply:
Yea it's weird. It's the auto formatter that did this.